src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js

```diff
@@ -53,10 +53,10 @@ define(['angular', 'application/interviewer/tdprInterviewerModule', 'application
         function activate() {
             $scope.clearTable();
             updateDate();
-            tdprSlotsService.getSlotsTimes().then(function(response) {
-                for(var i = 0; i < response.data.length; i++) {
-                    var startTime = response.data[i].startTime.slice(0,5);
-                    var endTime = response.data[i].endTime.slice(0,5);
+            tdprSlotsService.getSlotsTimes().then(function (response) {
+                for (var i = 0; i < response.data.length; i++) {
+                    var startTime = response.data[i].startTime.slice(0, 5);
```

Comment:
`0` and `5` are magic values as well - it would be good to extract constants here and create a new function which will call `slice`, since the same operation is performed in multiple places.